platform/ios/CHANGELOG.md

```diff
@@ -9,10 +9,12 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added a new `MGLMapSnapshotter` class for capturing rendered map images from an `MGLMapView`’s camera. ([#9891](https://github.com/mapbox/mapbox-gl-native/pull/9891))
 * Reduced the time it takes to create new `MGLMapView` instances in some cases. ([#9864](https://github.com/mapbox/mapbox-gl-native/pull/9864))
 * Added support for forced cache revalidation that will eliminate flickering that was sometimes visible for certain types of tiles (i.e., traffic tiles). ([#9670](https://github.com/mapbox/mapbox-gl-native/pull/9670), [#9103](https://github.com/mapbox/mapbox-gl-native/issues/9103))
+* Improved the performance of the SDK when parsing vector tile data used to render the map. ([#9312](https://github.com/mapbox/mapbox-gl-native/pull/9312))
 
 ### Styles
 
 * Added a new type of source, represented by the `MGLImageSource` class at runtime, that displays a georeferenced image. ([#9110](https://github.com/mapbox/mapbox-gl-native/pull/9110))
+* Setting a style using `MGLMapView`'s `styleURL` property now smoothly transitions from the previous style to the new style and maintains equivalent layers and sources along with their identifiers. ([#9256](https://github.com/mapbox/mapbox-gl-native/pull/9256))
```

Comment:
Does this mean it’s safe to maintain a reference to a MGLStyleLayer or MGLSource object across style changes?

Reply:
That was my understanding after reading https://github.com/mapbox/mapbox-gl-native/pull/9256. The copy here (and the copy in question in https://github.com/mapbox/mapbox-gl-native/pull/9919#discussion_r137413610) was taken from that PR.